tests/simulator/test_advance.py

```
@@ -86,6 +86,9 @@ def getHierarchy(
             strict=True,
         )
 
+        def S(x, x0, l):
```

Ruff (0.8.2)

Ambiguous variable name: `l`

(E741)